tools/RAiDER/cli/raider.py

```diff
@@ -225,6 +227,8 @@ def calcDelays(iargs=None):
     else:
         wm_bounds = aoi.bounds()
 
+    model.set_latlon_bounds(wm_bounds) # set the weather model bounds
```

Comment:
set the weather model bounds in the class itself so we don't have to pass them all over the place